src/DotNetWorker.Grpc/GrpcCollectionModelBindingData.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
+{
+    internal partial class GrpcCollectionModelBindingData : Microsoft.Azure.Functions.Worker.Core.CollectionModelBindingData
+    {
+        private readonly CollectionModelBindingData _modelBindingDataArray;
+
+        public GrpcCollectionModelBindingData(CollectionModelBindingData modelBindingDataArray)
+        {
+            _modelBindingDataArray = modelBindingDataArray;
+        }
+
+        public override Core.ModelBindingData[] modelBindingDataArray => _modelBindingDataArray.ModelBindingData.Select(
```

Comment:
Won't this LINQ expression be executed every time the property is accessed?. We should cache the result of this LINQ expression code locally and return that.  Perhaps you can execute this in the constructor?

Reply:
Same question for  `Content` prop of `GrpcModelBindingData`

Reply:
Updated